File: CSharp/Calculator/CalculatorException.cs
/*--------------------------------------------------------------------------*\
::
::  Copyright © 2023 Steffen Liersch
::  https://www.steffen-liersch.de/
::
\*--------------------------------------------------------------------------*/

using System;

namespace Liersch.Calculator;

sealed class CalculatorException : Exception
{
  public CalculatorException(string message) : base(message) { }
}

File: Python/calculator_exception.py
#:----------------------------------------------------------------------------
#:
#:  Copyright © 2023 Steffen Liersch
#:  https://www.steffen-liersch.de/
#:
#:----------------------------------------------------------------------------

class CalculatorException(Exception):
  def __init__(self, message: str):
    super().__init__(message)
